src/app/lightning/index.ts

```diff
@@ -1,3 +1,18 @@
-export * from "./payment-status-checker"
-export * from "./lookup-invoice-by-hash"
-export * from "./lookup-payment-by-hash"
+import { wrapAsyncToRunInSpan, wrapToRunInSpan } from "@services/tracing"
+import * as LookupInvoiceByHash from "./lookup-invoice-by-hash"
+import * as LookupPaymentByHash from "./lookup-payment-by-hash"
+import { PaymentStatusChecker as PaymentStatusCheckerFn } from "./payment-status-checker"
+
+// creates an object with all the functions of imported modules
+const dynamicWrapper = { ...LookupInvoiceByHash, ...LookupPaymentByHash }
```

Comment:
maybe the name below make more sense?

```suggestion
const allAsyncFunctions = { ...LookupInvoiceByHash, ...LookupPaymentByHash }
```

Reply:
Updated in the new PR